docs/content/content-blocks.md

```diff
@@ -110,6 +77,69 @@ My content
 ````
 `````
 
+### New Style Admonitions
+
+The admonition syntax above utilises the general [directives syntax](content:myst/directives).
+This is has the advantage of making it consistent with every other directive.
+However, a big disadvantage is that, when working in any standard Markdown editor (or the Jupyter Notebook interfaces),
+the text they contain will not render nicely as standard Markdown.
```

Comment:
@chrisjsewell just to clarify -- this is for compatibility with local previews for `editing` right (or jupyter notebook)? The usual "```" directive should now contain `md` -- it is just that editors won't syntax highlight them as they are interpreted as literal (with a highlight definition).

Reply:
> this is for compatibility with local previews for editing right (or jupyter notebook)?

Yep thats correct

> The usual "```" directive should now contain `md`

Not sure I get exactly what you mean by this?
But essentially, under the hood, it is conceptually different; because instead of passing the "raw" text to a sphinx directive (and getting back a docutils node), you are taking Markdown tokens and constructing the docutils nodes from that

Reply:
and yeh, as explained, I think its just a lot more new user friendly.
For example in this page the first admonition now correctly shows the URL links https://github.com/executablebooks/jupyter-book/blob/upgrade/myst-nb-0.9/docs/intro.md

Reply:
thanks @chrisjsewell I think I just added a suggestion to emphasis local previews to prevent possible confusion with parsing via `jupyter-book`